backend/src/main/java/com/ambacam/model/TypeRequete.java

```diff
@@ -0,0 +1,70 @@
+package com.ambacam.model;
+
+import org.hibernate.validator.constraints.NotEmpty;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.io.Serializable;
+
+@Entity
+public class TypeRequete implements Serializable {
```

Comment:
Please generate the serialVersionUID (put your mouse on the name of the class and then choose "add generate serial version ID")

Reply:
I'm using intellij and there ist no possibility to generate that.I think Intellij never use serialVersionUID 
In what way is the serialVersionUID important

Reply:
Here is a link that explains you how to activate and generate it:
https://netlicensing.io/blog/2012/12/25/intellij-idea-how-to-generate-serialversionuid/